alacritty.yml

```diff
@@ -640,9 +640,18 @@
 #
 # - `command`: Fork and execute a specified command plus arguments
 #
-#    The `command` field must be a map containing a `program` string and an
-#    `args` array of command line parameter strings. For example:
-#       `{ program: "alacritty", args: ["-e", "vttest"] }`
+#    The `command` field can be either only a string specifying the program, or
+#    a map of:
+#
+#    - `program` (required), string specifying the program
+#    - `args` (optional), array of command line parameter strings
+#    - `input` (optional), either VisibleText or AllText. If specified, then
+#      respectively either the visible text in the terminal or all the text
+#      including the scrollback is piped to the command's stdin.
+#    - `pipe_escape_sequences` (optional), if set to `true`, then Alacritty will also
+#      pipe ANSI escape sequences. This is slow so it should be left disabled when possible.
```

Comment:
This seems like a dangerous way to do configuration. What do we expect to happen when input isn't present but `pipe_escape_sequences` is?

Why was this split into two different fields when they're so tightly coupled together?

Reply:
Would using a pipe to combine them be more suitable? Like
```yml
input: VisibleText | EscapeSequences
```
This syntax is already used for keybindings and we could require having at least `VisibleText` or `AllText` present for `EscapeSequences` to be usable in the field.